Account types (#252)
trilitech/tezos-developer-docs: docs/smart-contracts/data-types/complex-data-types.md

Contracts can pass tickets to entrypoints to change which contract is in control of the ticket.
If contract A passes a ticket to contract B, contract A loses all access to the ticket.

> Contracts can pass tickets only to other contracts because the entrypoint must accept a ticket of the correct type; contracts cannot pass tickets to user accounts.

Reviewer: This is not true, see <https://tezos.gitlab.io/active/tickets.html#transferring>:
```suggestion
Contracts can pass tickets to other contracts via entrypoints accepting a ticket of the correct type; contracts can also pass tickets to user accounts.
```
